### DISCUSSION TITLE
Is Flectra still actively developed

### DISCUSSION THREAD

**Discussion by smartkid808** [flectra-hq/flectra#41] [General]
Hi,  I was wondering if Flectra is still being actively developed and what extras are included in the paid "pro" sub thats not in the community version.  I emailed them about a week ago but haven't heard back

**Comment by dmak** [comment]
I've downloaded latest Flectra [flectra_setup_3.0.20251205.exe](https://download.flectrahq.com/3.0/nightly/windows/flectra_setup_3.0.20251205.exe), and after installation I cannot add a bank statement 😭 There is no link or button for that 🤔
Also I am not sure how to import statements, and connect to real bank to fetch statements (should it be mentioned in [Account basics video](https://www.youtube.com/watch?v=0TyrTABBjEo)?)
Further comments are welcome.
<img width="1781" height="786" alt="image" src="https://github.com/user-attachments/assets/83f3316a-d728-4c9e-a2bb-55988aebe8b1" />

**Reply by smartkid808** [reply]
I think this project might be dead

**Reply by dmak** [reply]
Thanks. Sorry for out-of-topic, but what alternative software have you tried? I am looking into either Odoo ([cloud version / 1 free app](https://www.odoo.com/pricing)) or [Dolibarr](https://www.dolibarr.org/). My feature requirements are:
* VAT support (quarter / annual reporting)
* Account tree support (preferably with account codes).
* Clear/easy CSV/XLSX bank transaction import (preferably MT940 / CAMT.53) and export with account codes support.
